Dockerfile

```diff
@@ -127,7 +127,9 @@ RUN apt-get -qq update \
     libtbb2 libtbb-dev libdc1394-22-dev libopenexr-dev \
     libgstreamer-plugins-base1.0-dev libgstreamer1.0-dev \
     # scipy dependencies
-    gcc gfortran libopenblas-dev liblapack-dev && \
+    gcc gfortran libopenblas-dev liblapack-dev \
+    # faster-fifo dependencies
+    g++ cython3 && \
```

Comment:
Something weird happened in your rebase. These arent related commits

Reply:
That is very weird, and then all I did was amend the commit to have a different message and now it fixed